src/components/react-flow/base-node.tsx

```
@@ -0,0 +1,88 @@
+        "bg-card text-card-foreground relative rounded-md border",
+        "hover:ring-1",
+        // React Flow displays node elements inside of a `NodeWrapper`
+        // component, which compiles down to a div with the class
+        // `react-flow__node`. When a node is selected, the class `selected` is
+        // added to the `react-flow__node` element. This allows us to style the
+        // node when it is selected.
+        "in-[.selected]:border-muted-foreground",
+        "in-[.selected]:shadow-lg",
+        className,
+      )}
+      tabIndex={0}
```

Biome (2.4.14)

[error] The HTML element div is non-interactive. Do not use tabIndex.

(lint/a11y/noNoninteractiveTabindex)